app/src/components/textfield/TextField.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { forwardRef, Ref } from "react";
+import {
+  TextField as AriaTextField,
+  TextFieldProps as AriaTextFieldProps,
+} from "react-aria-components";
+import { css } from "@emotion/react";
+
+import { fieldBaseCSS } from "@phoenix/components/field/styles";
+import { SizingProps } from "@phoenix/components/types";
+export interface TextFieldProps extends AriaTextFieldProps, SizingProps {}
+
+const textFieldCSS = css`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```

Comment:
I'm not quite happy with this - I think it should be fit-content but not sure how to make form layouts nice quite yet.

Reply:
I tend to like 100% myself, and then control the width in the parent container via flex with siblings or grid cols

Reply:
I think you're right - that tends to be what I want.